nobl9/log_collection_schema.go

```diff
@@ -0,0 +1,30 @@
+package nobl9
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const logCollectionConfigKey = "log_collection_enabled"
+
+func schemaLogCollection() *schema.Schema {
+	return &schema.Schema{
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Description: "[Logs documentation](https://docs.nobl9.com/Features/direct-logs)",
+	}
+}
+
+func marshalLogCollectionEnabled(d *schema.ResourceData) *bool {
+	lData := d.Get(logCollectionConfigKey)
```

Comment:
In the previous version, you were checking if the value was provided. Could you give me more insights into why it was removed?

Reply:
Thanks